packages/terra-clinical-onset-picker/src/OnsetPicker.scss

```diff
@@ -12,4 +12,26 @@
     margin-bottom: 0.357em;
     margin-right: 0.357em;
   }
+
+  .precision {
+    min-width: 150px;
+  }
+
+  .granularity {
+    min-width: 100px;
+  }
+
+  .month {
+    min-width: 150px;
```

Comment:
When the granularity is Age and you switch between selecting Weeks and Months as the age unit, the width of the age value input changes.

Reply:
Yeah I just noticed this, I'll be adding a fixed width to the age input